site/guide/swap-documentation-templates.qmd

```diff
@@ -3,9 +3,10 @@ title: "Swap documentation templates"
 date: last-modified
 ---
 
-Swapping templates allows you to switch to a completely different template, upgrade to a more recent version of your current template, or make changes to both the template and template version at the same time.
+Easily swap between different versions of your model documentation or validation report templates within the {{< var vm_platform >}}. Switch to a completely different template, or apply another version of your current template.
```

Comment:
Re: "Easily swap": Still working on my review, but I think we should chat about our use of words like "easily", "convenient", or "simply" as they are usually removed by technical editors I have worked with. The problem is that these words are vague and sit somewhere outside of the realm of technical communication — what's easy to you might not be easy to me and they aren't measurable. What's worse, if a user tries something, can't perform a task, and then sees our claim that it's easy, it _really_ doesn't help. Best to avoid, I think?

Reply:
That makes sense — you'll have to forgive me, I spent the last half-decade in martech land and am still shaking some of it off 😆